src/locales/en/translation.json

```diff
@@ -0,0 +1,5 @@
+{
+    "login": "Login",
```

Comment:
it is good if we will give the capital letter for the key,because constant are gen are in capital letter

Reply:
Keys should not be in capital